third_party/opa/data-and-policies/sample_policies.rego

```diff
@@ -14,7 +14,7 @@ transform[action] {
 }
```

Comment:
Another condition needs to be checked in addition to those already there.  Check that the processing geography is not Turkey.

There should be a separate rule similar to this one but that indicates if the processing geography is Turkey then allow.  
I.e If purpose = Fraud Detection and role=data scientist and residency=Turkey and processing geography!=Turkey result = allow